test/quantization/test_quantize_script.py

```diff
@@ -1449,6 +1449,26 @@ def forward(self, x):
                    .check("aten::dequantize") \
                    .run(m.graph)
 
+
+    def test_qbatch_norm(self):
```

Comment:
can you move this to after `test_quantized_cat`? also can you use `self._test_op_impl` function here?